ops/deploy.yml

```diff
@@ -0,0 +1,64 @@
+- name: Get timestamp for repoo
+  hosts: localhost
+  tasks:
+    - shell: 'date +%Y%m%d%H%M%S.%5N'
+      register: current_run_timestamp
+    - set_fact: ansistrano_release_version = "{{ current_run_timestamp.stdout }}"
+
+- name: Permissions and folders
```

Comment:
all 3 "Permissions and folders" tasks are essentially the same with the exception of the path. could `with_items` be used here?